tangos/input_handlers/yt.py

```diff
@@ -88,3 +90,102 @@ def _load_halo_cat_without_caching(self, ts_extension, snapshot_file):
                                                 hubble_constant = snapshot_file.hubble_constant)
         cat_data = cat.all_data()
         return cat, cat_data
+
+
+class YtEnzoRockstarInputHandler(YtInputHandler):
+    patterns = ["RD????/RD????", "DD????/DD????"]
+    auxiliary_file_patterns = ["halos_*.bin"]
+
+    def load_timestep_without_caching(self, ts_extension, mode=None):
+        from yt.data_objects.particle_filters import add_particle_filter
+        if mode!=None:
```

Comment:
This should probably be `mode is not None` for style

Reply:
Fixed